x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/fingerprint.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCode } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { FieldsConfig, from, to } from './shared';
+import { TargetField } from './common_fields/target_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import {
+  FIELD_TYPES,
+  Field,
+  UseField,
+  SelectField,
+  ComboBoxField,
+  fieldValidators,
+} from '../../../../../../shared_imports';
+
+const fieldsConfig: FieldsConfig = {
+  fields: {
+    type: FIELD_TYPES.COMBO_BOX,
+    deserializer: to.arrayOfStrings,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.fingerprint.fieldNameField', {
+      defaultMessage: 'Fields',
+    }),
+    helpText: i18n.translate('xpack.ingestPipelines.pipelineEditor.fingerprint.fieldNameHelpText', {
+      defaultMessage: 'Fields that should be included in the fingerprint.',
```

Comment:
I think we can tighten this a bit.
```suggestion
      defaultMessage: 'Fields to include in the fingerprint.',
```